qcfractal/interface/models/common_models.py

```diff
@@ -200,11 +200,11 @@ def get_hash_index(self):
 
 class Citation(ProtoModel):
     """ A literature citation.  """
-    _acs_citation: str  # hand-formatted citation in ACS style. In the future, this could be bibtex, rendered to different formats.
-    _bibtex: Optional[Dict[str, str]]  # bibtex blob for later use with bibtex-renderer
-    doi: Optional[str]
-    url: Optional[str]
+    acs_citation: str = None  # hand-formatted citation in ACS style. In the future, this could be bibtex, rendered to different formats.
```

Comment:
`Optional[str]`

Reply:
Thanks for catching this. I'm clearly struggling with this particular model :P